storage/src/main/scala/ch/epfl/bluebrain/nexus/storage/routes/Routes.scala

```diff
@@ -85,16 +85,13 @@ object Routes {
     */
   def apply(
       storages: Storages[Task, AkkaSource]
-  )(implicit config: AppConfig, identities: DeltaIdentitiesClient[Task]): Route =
+  )(implicit config: AppConfig, authorizationMethod: AuthorizationMethod): Route =
     //TODO: Fetch Bearer token and verify identity
     wrap {
       concat(
         AppInfoRoutes(config.description).routes,
-        (pathPrefix(config.http.prefix) & extractToken) { implicit token =>
-          extractCaller.apply {
-            case Caller(config.subject.subjectValue, _) => StorageRoutes(storages).routes
-            case _                                      => failWith(AuthenticationFailed)
-          }
+        (pathPrefix(config.http.prefix) & validateUser) {
```

Comment:
This is just a thought I had, but I wonder if this makes more sense that you say `validUser` here

Now I've learned a bit about akka I wonder if this is suppose to read as:

- The request has a path prefix of `config.http.prefix`
- The request has a valid user

Not an issue at all just something I was thinking about